syntax = "proto3";

message User {
  string name = 1;
}

message UserId {
  uint64 id = 1;
}
